config/prod/TESLA-download-bucket.yaml

```diff
@@ -1,7 +1,14 @@
 # Provision a general S3 bucket or a Synapse External Bucket
 # http://docs.synapse.org/articles/custom_storage_location.html
-template_path: "remote/s3-bucket-v2.j2"
+template:
+  type: "http"
+  url: "https://{{stack_group_config.admincentral_cf_bucket}}.s3.us-east-1.amazonaws.com/aws-infra/v0.2.7/s3-bucket-v2.j2"
 stack_name: "TESLA-download-bucket"
+stack_tags:
+  Department: "CompOnc"
+  Project: "TESLA"
+  OwnerEmail: "james.eddy@sagebase.org"
+  CostCenter: "Other / 000001"
```

Comment:
Since `CostCenter` is "Other", there should be an OtherCostCenter (or similar, I forget the exact tag name) with the tag for TESLA.  Remember that if there *is* no cost center (e.g. no specific project), then it's OK to have "No Program / 000000".

Reply:
it was just a guess, do you have a better suggestion @brucehoff ?  otherwise maybe @jaeddy can suggest a more appropriate value?

Reply:
@zaro0508 I know of no program code for Tesla so please use "NO PROGRAM / 000000"